applications/nrf5340_audio/shifted_bin/revision.txt

```diff
@@ -0,0 +1,2 @@
+ble5-ctr-rpmsg_shifted.hex for normal b0n size
```

Comment:
I do not think we need a .txt file here. This information can be written in the doc (README.rst)

Reply:
Removed.